lib/autoversion/gitter.rb

```diff
@@ -22,18 +22,19 @@ def ensure_cleanliness!
     end
 
     def dir_is_clean?
-      # sum = @repo.status.untracked.length + @repo.status.added.length + @repo.status.changed.length + @repo.status.deleted.length
-      # if sum > 0
-      #   puts "untracked: #{@repo.status.untracked}"
-      #   puts "added: #{@repo.status.added}"
-      #   puts "changed: #{@repo.status.changed}"
-      #   puts "deleted: #{@repo.status.deleted}"
-      # end
+      sum = gitstatus_untracked_workaround.length +
+            @repo.status.added.length +
+            @repo.status.changed.length +
+            @repo.status.deleted.length
 
-      # sum == 0
+      if sum > 0
+        puts "untracked: #{gitstatus_untracked_workaround.join(', ')}"
```

Comment:
I think it's a bit too noisy to print these lines when incrementing a version. Instead I want to grab the raised exception in the Cli class and print a nice error message when a repository is not clean (maybe it can include specifics about the offending files). I can work on this after we merge this PR.